lib/helpers.js

```diff
@@ -60,8 +60,29 @@ exports.validateName = function (name) {
 };
 
 /**
+ * Check if relation has a different name from properties
+ * @param {Object} modelDefinition The model which has the relation
+ * @param {String} name The user input
+ * @returns {String|Boolean} The check result
+ */
+exports.checkRelationName = function (modelDefinition, name, callback) {
+  modelDefinition.properties(function(err, list) {
+    if (err) callback(err);
+    var conflict = list.some(function(property) { 
+      return property.name === name; 
+    });
+    if (conflict) {
+      callback('Same property name already exists: ' + name);
+    }
+    else {
+      callback(true);
+    }
+  });
+};
+
+/**
  * Get the root command name based on the env var
- * @returns {string}
+ * @returns {string} 
```

Comment:
Trailing space, please remove.

Most text editors can be configured to automatically remove trailing spaces, I find it personally best to turn on this option and don't have to worry about trailing spaces at all.